common/core/web/keyboard-processor/src/text/kbdInterface.ts

```diff
@@ -787,7 +795,36 @@ namespace com.keyman.text {
       outputTarget.restoreProperties();
     }
   
-    
+    /**
+     * `contextExOutput` function emits the character or object at `contextOffset` from the
+     * current matched rule's context. Introduced in Keyman 14.0, in order to resolve a
+     * gap between desktop and web core functionality for context(n) matching on notany().
+     * See #917 for additional detail.
+     * @alias       KNO
```

Comment:
Why `KNO`?  Is it for **n**otany**Output** or something like that?  Most of the shorthand/abbreviated forms are relatively simple to interpret, especially when referring to the TS-source 'long names'... but this one eludes me a bit.  I'd prefer something that matches a little more closely.

Reply:
> Is it for **n**otany**O**utput or something like that?

Yep. But feel free to suggest a better short name 😁. `KCXO`?

Reply:
`KCXO` sounds fine... I'm fine with `X` representing `notany`, as a whole representing _**c**_ontext _**(x)**_notany _**o**_utput.

Reply:
So `KCXO` is to get used in #3816 and added to help.keyman-staging.com/developer/engine/web/14.0/reference/interface ?